homeassistant/components/vconnex/const.py

```diff
@@ -0,0 +1,45 @@
+"""Constants for the Vconnex integration."""
+
+from __future__ import annotations
+
+DOMAIN = "vconnex"
+DOMAIN_NAME = "Vconnex"
+PROJECT_CODE = "HASS"
+
+PLATFORMS = ["switch"]
```

Comment:
Maybe import it from the switch module:
`from homeassistant.components.switch import DOMAIN as SWITCH_DOMAIN`

```suggestion
PLATFORMS = [SWITCH_DOMAIN]
```